src/blocks/content-helper/icons/published-link-icon.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/components';
+
+export const ViewsIcon = () => (
+	<SVG
```

Comment:
Please see https://css-tricks.com/accessible-svgs/#aa-example-2-standalone-icon-decorative and add the `aria-hidden` attribute as necessary.

Reply:
Added in next line.